web/src/components/starter-kits/StepInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { Step } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+export const StepInfo = (props: { step: Step; taskNames: string[] }): ReactElement => {
+  return (
+    <div className="flex-basis-100">
+      <div className="margin-top-05 margin-bottom-2">
+        <span className="border-2px border-secondary-vivid-dark text-secondary-vivid-dark radius-lg padding-x-1 padding-y-05">
+          <b>STEP {props.step.stepNumber}</b>
+        </span>
+      </div>
+      <Heading level={3} styleVariant="h4">
+        {props.step.name.replace("${OoS}", "")}
+      </Heading>
+      <p>{props.step.description}</p>
+      <b className="text-base-dark">{`(${props.step.timeEstimate})`}</b>
```

Comment:
[sand] Another opportunity to use `<strong>` rather than `<b>`. And it should semantically probably be inside the same paragraph as the description, so that screen readers don't read it as a separate element.